gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/snapshot/org.apache.gobblin.service.flowconfigsV2.snapshot.json

```diff
@@ -67,6 +67,11 @@
       "type" : "boolean",
       "doc" : "Return the compiled flow as a string. If enabled, the flow is not added.",
       "default" : false
+    }, {
+      "name" : "owningGroup",
```

Comment:
We may need to handle this in FlowSpecDeserializer/FlowSpecSerializer

Reply:
and also update mysql table schema with this field, and handle this field in MysqlSpecStore ?
Feel free to do this in other PR.

Reply:
I think the FlowSpecDeserializer/Serializer should be fine, since it reads the fields from `getConfigAsProperties` which I modified on `FlowSpec`. I'll take a look into seeing what needs to be changed in MySqlSpecStore though, good catch.